plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/components/inspector.tsx

```diff
@@ -4,29 +4,117 @@
 import { InspectorControls } from '@wordpress/block-editor';
 import { BlockEditProps } from '@wordpress/blocks';
 import { __ } from '@wordpress/i18n';
-import { PanelBody, ToggleControl } from '@wordpress/components';
+import {
+	Flex,
+	FlexItem,
+	PanelBody,
+	RadioControl,
+	ToggleControl,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import { Attributes } from '../types';
+import RatingStars from './rating-stars';
+import type { Attributes } from '../types';
+
+function MinimumRatingLabel( {
+	stars,
+	ariaLabel,
+}: {
+	stars: number;
+	ariaLabel: string;
+} ) {
+	return (
+		<Flex
+			title={ ariaLabel }
+			aria-label={ ariaLabel }
+			justify="flex-start"
+			gap={ 1 }
+		>
+			<FlexItem>
+				<RatingStars stars={ stars } />
+			</FlexItem>
+			<FlexItem>{ __( '& up', 'woocommerce' ) }</FlexItem>
+		</Flex>
+	);
+}
 
 export const Inspector = ( {
 	attributes,
 	setAttributes,
 }: Pick< BlockEditProps< Attributes >, 'attributes' | 'setAttributes' > ) => {
-	const { showCounts } = attributes;
+	const { showCounts, minRating } = attributes;
+
+	function setCountVisibility( value: boolean ) {
+		setAttributes( {
+			showCounts: value,
+		} );
+	}
+
+	function setMinRating( value: string ) {
+		setAttributes( {
+			minRating: value,
+		} );
+	}
```

Comment:
Nit pick but I don't see the benefit of defining the function and only using it once. The body of the function is also very short. Having `setAttributes` call inside `onChange` callback makes it easier for me to understand the callback without jumping to the definition.

Reply:
the benefit is mostly the code structure:

<img width="248" alt="image" src="https://github.com/user-attachments/assets/69128006-d9bb-4e7b-b168-2681032fd569">

I do like `onChange={ setCountVisibility }`, but it's just a detail. We can use inline functions straight in the prop value. Lmk if we should do it, please

Reply:
> Lmk if we should do it, please

I don't know. We don't have a rule on this. It's up to you :). I don't have a  strong opinion on coding style as everyone is different. That's why I marked this one as optional.